src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/srm/algebras/BDDAlgebra.cs

```diff
@@ -303,7 +308,7 @@ private BDD MkNot_rec(BDD a)
                 return neg;
 
             neg = a.IsLeaf ?
-                MkBDD(CombineTerminals(BoolOp.NOT, a.Ordinal, 0), null, null) : //muti-terminal case
+                MkBDD(CombineTerminals(BoolOp.NOT, a.Ordinal, 0), null, null) : // multi-terminal case
```

Comment:
Is the "Mk" prefix short for "Make"?  Could we spell it out, or use "Create" or something a little more standard?

Reply:
Yes, it is short for Make. I have no strong preference between Make or Create. If Create feels least surprising to you that could be a good choice.

The prefix is also used in the IBooleanAlgebra interface for logical operations, but unlike `MkBDD` those aren't guaranteed to be "making" anything. Rather, depending on the algebra in question they might be operating on value types (BV64Algebra) or sometimes just returning something from a cache (BDDAlgebra), so the prefix might be a bit misleading. The prefix also isn't used quite consistently, e.g. the following line uses the BDDAlgebra, but the additional function `ShiftLeft` supplied by the BDDAlgebra doesn't use the prefix.
```
_IgnoreCaseRel_inv = _solver.MkAnd(inv_table, _solver.MkNot(_solver.ShiftLeft(tr_I_withdot_BDD, 16)));
```
Any suggestions here? One option would be to drop the Mk prefix and just have that line be:
```
_IgnoreCaseRel_inv = _solver.And(inv_table, _solver.Not(_solver.ShiftLeft(tr_I_withdot_BDD, 16)));
```

Reply:
> One option would be to drop the Mk prefix and just have that line be

I like that (I think :)